### ISSUE TITLE
Is there any way to remove metadata from images

### ISSUE THREAD

**Issue by bigint** [willnorris/imageproxy#372]
Is there any way to remove metadata like GPS location etc from an image?

**Comment by willnorris** [comment]
No, not currently.  I believe the exif library I'm currently using (https://github.com/rwcarlsen/goexif) only supports decoding exif data.  It looks like https://github.com/dsoprea/go-exif also supports encoding exif, but I'm not sure I love the idea of doing that.  If the goal is location privacy, this isn't really going to do a lot, since imageproxy makes no effort to hide the URL of the source image, which would still contain the location data.  If the goal is file size, how large is the location data compared to the image itself?  I would guess it's just lat/lon coordinates, altitude, and a timestamp or something?
